_posts/2017-10-25-myget-by-the-numbers-2017.markdown

```diff
@@ -0,0 +1,76 @@
+
+The shared tenant has its primary deployment in the West Europe region, and a secondary in US East. We use Azure Cloud Services to deploy both Web and Worker roles, and apply auto-scaling policies on both to ensure the service scales as needed.
+
+On average, the shared tenant has 4 Web roles and 4 worker roles deployed in each region, and scales in and out multiple times (minimum 2 instances per role, up to 8 during peak moments). Judging by the observed peak moments, all of us restore most packages in the morning (which is multiple times a day when running a global service).
+
+## Reliability
+
+Being developers ourselves, we know how frustrating it is when a service you use on a daily basis goes down. Even though we cannot offer an official SLA just yet, we can demonstrate historical uptime data on our [status page](https://www.myget.org/status). We invite you to go take a look if interested (and it lists uptime data for other endpoints too). 
+
+The most interesting data point for package consumers is likely the following one: package download uptime. We're happy to share that, in 2017, we're currently on track to exceed the 99.99% uptime goal we set ourselves!
+
+Currently, the package download endpoint has an uptime percentage of **99.992%**! In other words, the endpoint has been down for 35 minutes in 2017, spread out over 5 different days, so that's 5 days this year with an average interruption of less than 7 minutes. If you didn't do a restore during those brief interruptions, you likely haven't noticed this interruption at all. If you did, we're very sorry, and please know that we'll continue to work hard to further improve on this metric.
+
+## Storage, traffic and bandwidth
+
+The below numbers include all supported package types.
+
+Our shared tenant currently hosts **over 20 million packages** (in comparison, that's 20x nuget.org, or 40x npmjs.com!), consuming **770GB of blob storage**. This does not include back-ups to cover for human error, and does not include metadata extracted during upload to facilitate querying. That is significant growth compared to July last year, when we hosted 1 million packages in total. 
+
+**On a monthly basis, over 550K packages are pushed to this tenant**, with an average success rate of **98.05%**. Success means: the package actually ends up on the feed. So 1.95% of packages are rejected because they fail server-side validation.
+Wednesday and Thursday seem to be the most productive days of the week with around **45K packages pushed on a single day**!
+
+In terms of bandwidth, MyGet's shared `www.` tenant serves around **10M requests a day** with an **average response time of 93.58ms**, accounting for **220GB egress, and 40GB ingress per day**. On a monthly basis, MyGet tenants generate **8.7TB egress traffic**!
```

Comment:
Did we include KeyCDN in egress numbers? If not, that's +/- 190 GB extra on top per day.

Reply:
Nope, did not include CDN egress yet, good one!